Resources/Prototypes/Entities/Structures/Power/chargers.yml

```diff
@@ -81,10 +82,10 @@
 - type: entity
   name: wall recharger
   id: WallWeaponCapacitorRecharger
-  parent: PowerCellRecharger
+  parent: WeaponCapacitorRecharger
```

Comment:
why this change?

Reply:
It didn't charge weapons like it was intended to